### ISSUE TITLE
Feature request: Run in Background (Mobile)

### ISSUE THREAD

**Issue by trymeouteh** [localsend/localsend#275]
### Is your feature request related to a problem? Please describe.

To have the app run in the background on mobile with a silent notification to indicate the app is running in the background,

### Describe the solution you'd like

To have the app run in the background and to have a button in the receive and send menu to start and stop the service from running in the background.

**Comment by sergd88** [comment]
I really like that the computer application can run in the background and automatically open the file receive window when file transfer request received. I don't know how difficult or possible it is, but I would like to have this behavior on my phone as well, where I can simply minimize the app and the file receive window would automatically open when file transfer request received. Later, a corresponding toggle switch could be added to the notification panel ( [#310](https://github.com/localsend/localsend/issues/310)) to enable/disable the app service running in the background. Also, so that the application, or more specifically the service, could autostart with system ([#276](https://github.com/localsend/localsend/issues/276)). And it would also be ideal, if the file receive request could also come through when the phone is locked and upon unlocking, the file transfer window would also automatically appear.

**Comment by k-svn** [comment]
I agree with this. It would be nice to have this service running in the background.

**Comment by Crunchbits** [comment]
I agree, instead of manually opening the app it would be so seamless to just get a notification on mobile when a file is being sent.

**Comment by programmerlexi** [comment]
I think having a quick settings tile for toggling service mode would be quite nice.
